hca2scea-backend/test_hca2scea.py

```diff
@@ -24,7 +24,7 @@ def test_hca2scea_characteristic(self):
             with self.subTest(spreadsheet="test/golden/" + list(arguments_df['spreadsheet'])[i]):
                 arguments = arguments_df.loc[arguments_df['spreadsheet'] == os.path.basename(spreadsheet)]
                 output_dir = self.run_tool(spreadsheet, arguments)
-                self.check_output(output_dir, spreadsheet)
+                self.check_output(os.path.join("test/golden",output_dir), spreadsheet)
```

Comment:
What is the purpose of this change?
You should not generate anything in the "golden" directory.